.markdownlint.yml

```diff
@@ -6,5 +6,7 @@ MD033: false
 
 MD024: false
 
+MD001: false
```

Comment:
what does this do?

Reply:
It suppresses some of  [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli)'s rules. This one in particular suppresses the header increments:
`MD001/heading-increment/header-increment Heading levels should only increment by one level at a time [Expected: h2; Actual: h4]`

I think it's easier to read when the headers are more prominent than it's subheaders